purchase_warn_message/__manifest__.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2020 ForgeFlow S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Purchase Warn Message",
+    "summary": """
+        Add a popup warning on purchase to ensure warning is populated""",
+    "version": "13.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "ForgeFlow, Odoo Community Association (OCA)",
+    "website": "www.github.com/OCA/stock-logistics-workflow.git",
+    "depends": ["purchase"],
```

Comment:
```suggestion
    "depends": ["purchase", "account_invoice_warn_message"],
```

Reply:
It is not required to be dependency of account_invoice_warn_message. Modules are independant, aren't they?

Reply:
Yes, but... the common sense says that if you want a warning in sale, and when installing sale your are installing account, then when you are installing this module, it may be implicit you want the warning in account (account invoice), because invoicing your sales are part of the sale workflow.

Reply:
sorry, I was thinking in sale, but the same argument applies for purchase.

Reply:
Maybe they are related on functionalities, but not on dependencies nor requirements, so I would like to skip account_invoice_warn_message dependency.